scripts/generate-translations.ts

```diff
@@ -0,0 +1,36 @@
+import * as fs from 'fs';
+import * as rimraf from 'rimraf';
+import { translations } from '../projects/storefrontlib/src/translations/index';
+
+const libDist = './dist/storefrontlib/';
+const translationsDist = libDist + 'translations/';
+function createDir(dir) {
+  if (!fs.existsSync(dir)) {
+    fs.mkdirSync(dir);
+  }
+}
+const getLangDir = lang => translationsDist + lang + '/';
+const getFileName = (lang, namespace) =>
+  getLangDir(lang) + namespace + '.' + lang + '.json';
```

Comment:
Maybe use here template string for concatenation. Should be more readable.

Reply:
:+1: